tests/gatys_et_al_2017/test_loss.py

```diff
@@ -80,6 +80,28 @@ def test_gatys_et_al_2017_style_loss(subtests):
         assert style_loss.score_weight == pytest.approx(1e3)
 
 
+def gatys_et_al_2017_guided_style_loss(subtests, content_guides):
```

Comment:
I didn't see this before, but if I'm not mistaken we never use the actual guides. Couldn't we do this at the beginning of the test?

```python
regions = tuple(f"region{idx}" for idx in range(3))
```

and test everything with that? IMO we don't need a fixture for that.

Reply:
Here we do not need the actual guides yet. However, we will need them later for `guided_nst`. Therefore I wanted to introduce the guides here to use these fixtures consistently for the guides.